k8s-scheduler/src/main/java/com/vmware/dcm/Scheduler.java

```diff
@@ -180,6 +188,11 @@ void scheduleAllPendingPods(final IPodToNodeBinder binder) {
         return podsToAssignUpdated;
     }
 
+    public void setScopeOn() {
```

Comment:
Leave a TODO above this method to make this part of the Scheduler's initial configuration. You don't need to make the change yourself, but just leave a note so we don't forget.